Renaming Agent's methods (#3557)
Unity-Technologies/ml-agents: docs/Learning-Environment-Create-New.md

### Initialization and Resetting the Agent

When the Agent reaches its target, its episode ends and the `OnEpisodeBegin()`

> function moves the target to a random location. In addition, if the Agent rolls

Reviewer: ```suggestion
method moves the target to a random location. In addition, if the Agent rolls
```

### Initialization and Resetting the Agent

When the Agent reaches its target, its episode ends and the `OnEpisodeBegin()`
method moves the target to a random location. In addition, if the Agent rolls

> off the platform, the reset function puts it back onto the floor.

Reviewer 2: `the reset function` -> `the OnEpisodeBegin() method`

> off the platform, the reset function puts it back onto the floor.

Author: ```suggestion
off the platform, the `OnEpisodeBegin()` method puts it back onto the floor.
```

}
```

The final part of the Agent code is the `Agent.OnActionReceived()` method, which

> receives the decision from the Brain and assigns the reward.

Reviewer: ```suggestion
receives the actions from the Brain and assigns the reward.
```
